### PR TITLE
installing minikube and tackle with action

### PR DESCRIPTION
**Pull request by JaydipGabani**

### CODE REVIEW

**Review by jmontleon — APPROVED** [review]

File: get-minikube/action.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Get minikube cluster
```
Comment (djzager) [inline get-minikube/action.yml:1]: I think we should move these two actions into a top-level `actions/` directory.

A follow-up PR would probably move the workflows out into a top-level `workflows/` (whatever doesn't get broken into actions).

File: install-tackle/action.yml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            value: ${{ inputs.tackle-addon-admin-image }}
+          - name: RELATED_IMAGE_ADDON_WINDUP
+            value: ${{ inputs.tackle-addon-windup-image }}
+      EOF
+  - name: Waiting for 60s for operator pod to come up 
+    uses: jakejarvis/wait-action@master
+    with:
+      time: '60s'
+  - name: Get pods in namespace
+    run: kubectl get pods -n konveyor-tackle
```
Comment (djzager) [inline install-tackle/action.yml]: You could specifically wait, something like:

```
kubectl --context dest --namespace konveyor-tackle wait --for=condition=ready pod --selector=name=tackle-operator --timeout=180s
```

**Review by djzager — COMMENTED** [review]
This is really great to see. I believe these actions will need to go in tackle2-operator via `make` or `/hack/scripts.sh`.

Thanks for putting it together.

File: get-minikube/action.yml (hunk 1 above)
Comment (mundra-ankur) [reply get-minikube/action.yml:1]: We can also move actions to `.github/actions` directory next to the GitHub Workflow's `.github/workflows` if it make sense

Comment (JaydipGabani) [reply get-minikube/action.yml:1]: About moving actions to top-level dir. We would need to do something like `actions/<action-name>/action.yml` and the same goes if we decide to move it under `.github/actions`. I tried `actions/<action-name>.yml` and it doesn't work.

Comment (mundra-ankur) [reply get-minikube/action.yml:1]: Yeah, I guess you are right, under the top-level directory you will need to add `<action-name>/action.yml`, be it `actions/` or `.github/actions/`

**Pull request closed without merging**